scripts/dbsetup.sql

```diff
@@ -68,3 +76,10 @@ CREATE TRIGGER set_timestamp_healthit_products
 BEFORE UPDATE ON healthit_products
 FOR EACH ROW
 EXECUTE PROCEDURE trigger_set_timestamp();
+
+
+CREATE or REPLACE VIEW org_mapping AS
```

Comment:
it looks like this doesn't include a trigger for deletes. see this: https://stackoverflow.com/questions/21301391/how-to-delete-automatically-all-reference-rows-if-parent-row-get-deleted-in-mysq

Reply:
Added the cascade delete -- appears to work 
![image](https://user-images.githubusercontent.com/3967953/76165794-4c05e280-6130-11ea-8486-6a138d2ea2e5.png)